lib/govuk_app_config/govuk_error/raven_delegator.rb

```diff
@@ -0,0 +1,17 @@
+module GovukError
+  class RavenDelegator < SimpleDelegator
+    def should_capture=(closure)
+      # rubocop:disable Style/RedundantSelf
```

Comment:
Oh how come you needed this?

Reply:
Ah, I needed it in an earlier iteration of the code, but have just removed it and it works fine :) good spot.